src/textual/widgets/_tabbed_content.py

```diff
@@ -151,37 +196,37 @@ def validate_active(self, active: str) -> str:
             Value of `active`.
 
         Raises:
-            ValueError: If the active attribute is set to empty string.
+            ValueError: If the active attribute is set to empty string when there are tabs available.
         """
-        if not active:
-            raise ValueError("'active' tab must not be empty string.")
+        if not active and self.get_child_by_type(ContentSwitcher).current:
+            raise ValueError(f"'active' tab must not be empty string.")
```

Comment:
Superfluous f string?

Reply:
Good catch, likely a slip of the thinger or a hangover from a version that said more.